specifications/api/swagger.yaml

```diff
@@ -379,6 +379,26 @@ x-stakePoolId: &stakePoolId
   example: pool1wqaz0q0zhtxlgn0ewssevn2mrtm30fgh2g7hr7z9rj5856457mm
   description: A unique identifier for the pool.
 
+x-drepKeyHash: &drepKeyHash
+  type: string
+  format: bech32
+  example: drep_vkh1wqaz0q0zhtxlgn0ewssevn2mrtm30fgh2g7hr7z9rj5856457mm
+  description: DRep's key hash.
+
+x-drepScriptHash: &drepScriptHash
+  type: string
+  format: bech32
+  example: drep_script1wqaz0q0zhtxlgn0ewssevn2mrtm30fgh2g7hr7z9rj5856457mm
+  description: DRep's key hash.
+
+x-drep: &drep
+  nullable: false
+  oneOf:
+    - <<: *drepKeyHash
+      title: DRep's key hash
```

Comment:
Maybe this title attribute could be moved to the corresponding `x-drep*Hash` close to other attributes?

Reply:
done